### ISSUE TITLE
Generate MARC records for DRS

### ISSUE THREAD

**Issue by jimk-bdrc** [buda-base/drs-deposit#100]
When Elie has created an API to generate a MARC XML record, create a script to generate records for Harvard DRS.

**Comment by jimk-bdrc** [comment]
jimk:> Does bdrc.io have a REST API for doing this, in the same way that tbrc does?

@eroux: I'm not sure what you mean by "this" here? Do you mean just exporting
a MARC record or more than that?

jimk:Yes, I do mean exporting a MARC XML record.

jimk:> (I'd prefer to use bdrc.io, because I believe there are some deficiencies in tbrc.org's MARC records (such as language - everything is TIB, even the FPL and Cambodia works).

eroux:Indeed, the MARC export of BUDA should be better... when it works
again! It shouldn't be too much work though, probably just 3 or 4
hours

jimk:> Even just an API to provide one work at a time would be ok for us (we just call it 14000 times), or if you wanted to provide a batch facility where we upload a list of RIDS, either way works fine for us.

eroux: I think the standard API should be good, just make a 2s pause between
each query please

**Comment by jimk-bdrc** [comment]
Waiting on @eroux - but no pressure, we might not be able to send these to Harvard for many weeks.

**Comment by jimk-bdrc** [comment]
See https://github.com/buda-base/lds-pdi/issues/204

**Comment by jimk-bdrc** [comment]
Waiting on #276
Related to #103

**Comment by jimk-bdrc** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Next DRS](https://trello.com/c/UqCp4aYg/10-next-drs)

**Comment by jimk-bdrc** [comment]
First 50 in QA sent to Corinna Baksik #117

**Comment by jimk-bdrc** [comment]
No longer needed - we generate metadata at build time for DRS3.
